osmosis/assetlist.json

```diff
@@ -16314,6 +16314,34 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/ashLAB.png"
         }
       ]
+    },
+    {
+      "description": "Cosmos Airdrop Chat",
+      "extended_description": "Official Token of the Cosmos Airdrop community",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/cac",
+          "exponent": 0
+        },
+        {
+          "denom": "CAC",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "address": "osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8",
+      "base": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/cac",
+      "name": "Cosmos Airdrop Chat",
+      "display": "CAC",
+      "symbol": "CAC",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/CAC.svg"
```

Comment:
the svg looks bad. Can you just use the png instead?

Reply:
Updated